lib/avalon/bib_retriever/MARC21slim2MODS3-5-avalon.xsl

```diff
@@ -1414,7 +1415,7 @@ Revision 1.02 - Added Log Comment  2003/03/24 19:37:42  ckeith
 		<!-- language 041 -->
 		<xsl:variable name="controlField008-35-37"
 			select="normalize-space(translate(substring($controlField008,36,3),'|#',''))"/>
-		<xsl:if test="$controlField008-35-37">
+		<xsl:if test="$controlField008-35-37 != 'N/A'">

```

Comment:
Should this test be in the the variable declaration? Something like this on line 1417:
<xsl:variable name="controlField008-35-37"
			select="normalize-space(translate(substring($controlField008,36,3),'|#','')) != 'N/A' "/>

I'm afraid that putting the test in lin 1418 will lead to false positives (when the field isn't found, field!='N/A' will be true).